base_continent/README.rst

```diff
@@ -0,0 +1,81 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+==============
+base_continent
+==============
+
+This module extends the functionality of 'base' module to support continents
+and to allow you to link each country with a continent
```

Comment:
I have a functional question about this module:
What's the difference between this module and the Odoo standard with the `res.country.group` model (Sales/Configuration/Contact/localization/Country Group).
Into this standard model, you can create a group of country (so a continent or something else) and add many countries as needed.
It is not more simple to keep the standard instead of create this new module?